config/more_samples-glyf_colr_1.py

```diff
@@ -137,6 +134,41 @@ def _sample_composite_colr_glyph():
         glyph_name=glyph_name, accessor="o", advance=_UPEM, glyph=pen.glyph(), colr=colr
     )
 
+def _gradient_stops_repeat(stops, accessor_char):
```

Comment:
if `stops` is always a tuple of 2 items you may as well unpack the parameters as `stop1` and `stop2`
otherwise turn `stops` into an open-ended `List[Tuple[float, str]]` where the `str` is the color name, so you can add more than 2 stops

Reply:
Made into separate stops, 2 are enough for this test.